### PR TITLE
ci: maintained and stable mariadb version (lts)

### PR DESCRIPTION
**Pull request by grooverdan**
Also use MARIADB env names and the healthcheck.sh included in the container.

### CODE REVIEW

File: .github/workflows/continuous-integration.yml [hunk 1]
```diff
@@ -190,7 +190,7 @@ jobs:
           - "default"
           - "3@dev"
         mariadb-version:
-          - "10.9"
+          - "lts"
```
Comment (SenseException) [inline .github/workflows/continuous-integration.yml]: Isn't lts a placeholder for the last supported lts? When MariaDB changes the actual version behind "lts" doesn't ORM support it automatically.

Comment (grooverdan) [reply .github/workflows/continuous-integration.yml]: > Isn't lts a placeholder for the last supported lts?

yes.

> When MariaDB changes the actual version behind "lts" doesn't ORM support it automatically.

You'd hope so, but that hope need to be enforced by the reality of CI.

Most important - 10.9 is out of [maintenance](https://mariadb.org/about/#maintenance-policy) and should no longer be used.

Comment (derrabus) [reply .github/workflows/continuous-integration.yml]: I'm fine with switching to maintained MariaDB versions in CI, but I agree that it should remain in our control which releases we actually test against. I don't think we should use the `lts` tag.

Comment (grooverdan) [reply .github/workflows/continuous-integration.yml]: done - 11.4

**Review by derrabus — APPROVED** [review]

**Review by SenseException — APPROVED** [review]

**Comment by grooverdan** [comment]
Thanks!

**Pull request merged**